seeds/web-demo/vite.config.ts

```diff
@@ -6,4 +6,4 @@ import Kits from "./build/graph.js";
 export default defineConfig({
   assetsInclude: ["**/*.graph"],
   plugins: [preact(), Kits(["@google-labs/llm-starter"])],
-});
+});
```

Comment:
@Chizobaonorh this is what caused the failure. The missing extra line break at the end of the file.